src/app/pages/search-character/search-character.component.ts

```diff
@@ -47,8 +47,8 @@ export class SearchCharacterComponent {
           this.charactersList = res.results.slice(0, 5);
           this.characterResultCounter = res.counter;
         },
-        error: () => {
-          this.charactersList = [];
+        error: (err) => {
```

Comment:
- the Callback error is retrieved but never used

 ```suggestion
        error: () => {
```